pkg/platform/kube/functionres/lazy.go

```diff
@@ -953,12 +954,20 @@ func (lc *lazyClient) populateIngressConfig(labels map[string]string,
 	for _, httpTrigger := range functionconfig.GetTriggersByKind(function.Spec.Triggers, "http") {
 
 		// set annotations
-		meta.Annotations = httpTrigger.Annotations
+		if httpTrigger.Annotations != nil {
+			meta.Annotations = httpTrigger.Annotations
+		}
 
 		// ignore any other http triggers, validation should catch that
 		break
 	}
 
+	if function.Status.State == functionconfig.FunctionStateScaledToZero {
+
+		// TODO change to get from known dlx services
+		meta.Annotations["nginx.ingress.kubernetes.io/default-backend"] = fmt.Sprintf("%s-dlx", function.Namespace)
```

Comment:
this won't work in the cloud systems - they are prefixed by release name.
add a label or something to the dlx service, use that to find it